src/components/Forms/LoginForm/index.jsx

```diff
@@ -0,0 +1,61 @@
+// hooks
+import { useState } from 'react';
+// styled components
+import { Form, Input, Label, Button, FormField } from '../styles.js';
+// complex components
+import SuccessNotifier from '../../SuccessNotifier';
+// constants
+import { LOGIN_FIELDS, LOGIN_INIT_STATE } from '../../../constants/formInfo';
+// validation
+import { validateLogin, loginFailed } from '../../../helpers/formValidation';
+
+const LoginForm = () => {
+    // field state
+    const [formData, setFormData] = useState(LOGIN_INIT_STATE);
+    // form state
+    const [errors, setErrors] = useState({});
+    const [success, setSuccess] = useState(false);
+
+    const handleInputChange = ({ target }) =>
+        setFormData({ ...formData, [target.name]: target.value });
+
+    const handleSubmit = event => {
+        event.preventDefault();
+
+        const validationErrors = validateLogin(formData);
+        setErrors(validationErrors);
+
+        if (loginFailed(validationErrors)) return;
+
```

Comment:
combine in one function validateLogin (which return {error, data})

Reply:
👍